src/components/CustomerQuota/ItemsSelection/ItemContent.tsx

```diff
@@ -62,13 +55,6 @@ export const ItemContent: FunctionComponent<{
           <ItemMaxUnitLabel unit={unit} maxQuantity={maxQuantity} />
         </AppText>
       )}
-      {descriptionAlert && descriptionAlert.length > 0 && (
-        <ShowAddonsToggle
-          descriptionAlert={descriptionAlert}
-          toggleIsShowAddons={showAddonsToggle}
-          isShowAddons={showAddons}
-        />
-      )}
```

Comment:
For now, yea. But this can be easily adapted to other `Item...` components! I put this away from `ItemContent` so that I don't pass too many props to `ItemContent`